adapter/internal/api/apis_impl.go

```diff
@@ -164,7 +146,6 @@ func ProcessMountedAPIProjects() (err error) {
 
 func validateAndUpdateXds(apiProject model.ProjectAPI, override *bool) (err error) {
 	apiYaml := apiProject.APIYaml.Data
-	apiProject.OrganizationID = config.GetControlPlaneConnectedTenantDomain()
```

Comment:
has been moved to https://github.com/wso2/product-microgateway/pull/2635/commits/14f55217da9e8884511a9cd8b071bfefdb57ab03#diff-d11ccea982a30d875a630f5024fcec9e09d72f24b3f5c59bb18c96e90e39ee9eR94